packages/react-native-windows-init/README.md

```diff
@@ -15,9 +15,14 @@ npx react-native-windows-init --language cpp
 Options:
 | option      | description                                    | type                                             |
 |-------------|------------------------------------------------|--------------------------------------------------|
-| --help      | Show help                                      | [boolean]                                        |
-| --version   | The version of react-native-windows to use.    | [string]                                         |
-| --namespace | The native project namespace.                  | [string]                                         |
-| --verbose   | Enables logging.                               | [boolean]                                        |
-| --language  | Which language the app is written in.          | [string] [choices: "cs", "cpp"] [default: "cpp"] |
-| --overwrite | Overwrite any existing files without prompting | [boolean]                                        |
+| `--help`      | Show help                                      | [boolean]                                        |
+| `--version`   | The version of react-native-windows to use.    | [string]                                         |
+| `--namespace` | The native project namespace.                  | [string]                                         |
+| `--verbose`   | Enables logging.                               | [boolean]                                        |
+| `--language`  | Which language the app is written in.          | [string] [choices: "cs", "cpp"] [default: "cpp"] |
+| `--overwrite` | Overwrite any existing files without prompting | [boolean]                                        |
+| `--noTelemetry` | Disables sending telemetry that allows analysis of failures of the react-native-windows CLI | [boolean] |
```

Comment:
Would be great if we could align this to `--no-telemetry` like is used for `run-windows`. `yargs` does have support for negation, of defining a `telemetry` option and having `--no-telemety` or `--telemetry false` work automatically. Not sure how that shows up in the help options though,

Reply:
Along the lines of designing the opt-out interface:  While I get that this may need to align with the telemetry approach in other MSFT open source projects, I'd prefer an environment variable that allows me to set this flag once and opt out permanently on my machine without remembering to issue a command every time.  It looks like from skimming this review that setting AGENT_NAME disables the telemetry (presumably to avoid capturing events on Azure DevOps build machines, I'm guessing?), but something documented and supported would be preferable.

If an environment variable is added, it should ideally be documented in the --help text as well, IMHO.

Reply:
yes, it should be --no-telemetry, this is a typo, fixed thanks!